x/supplier/client/cli/tx_stake_supplier_test.go

```diff
@@ -38,135 +38,219 @@ func TestCLI_StakeSupplier(t *testing.T) {
 		fmt.Sprintf("--%s=%s", flags.FlagFees, sdk.NewCoins(sdk.NewCoin(net.Config.BondDenom, sdkmath.NewInt(10))).String()),
 	}
 
+	defaultConfig := `---
```

Comment:
Should this not be a json string? or are we using yaml, if so please update mentions of json to yaml and in the parsing use yaml tags not json ones